src/components/resources.js

```diff
@@ -187,7 +187,7 @@ function Resources(props) {
               if (array.indexOf(x) === -1) array.push(x);
             });
           });
-          return array.map((x, i) => (
+          return array.sort().map((x, i) => (
```

Comment:
We should use meaning full variable name instead of 'x/i'. So that every developer can read and easy to understand the code. Code minify already doing sort our code during production build.

@nikita-11

Reply:
@aasimafridi44 We can take these changes separately for sure. Please review the changes that current PR involves. Addition of GA Trackers.